### ISSUE TITLE
How to Do Packaging and Publishing?

### ISSUE THREAD

**Issue by ssaarela** [finnair/v-validation#11]
How to do packaging and publishing?

Currently I'm publishing packages one at a time, but publishing moment after core fails as the tag exists already. Also it's easy to forgot updating moment->core dependency version. How do we publish/tag both packages at once?

Then there's github releases, I suppose just pushing a tag doesn't do a proper github release, does it? Unless it does, how should those be done in relation to yarn publish? Before or after?

**Comment by ssaarela** [comment]
Any input on this @paazmaya @eostman @N1cc3 @Jsemi ?

Also, should we setup e.g. a Slack workspace for discussion and coordination? I'd prefer asking stupid questions in a private chat 😄

**Comment by ssaarela** [comment]
The project-which-this-library-was-extracted-from now a) compiles and b) tests succeed with the latest 0.1.3 release. What I did was

1. Publish core
2. Fix moment->core dependency version
3. Publish moment ignoring the "fatal" error about duplicate tag
4. Deleted the tag from local git
5. Pushed to github
6. Performed release in github

There's a lot of room for error in this process and I feel that I'm probably doing something wrong as it shouldn't be this complicated - nor should it require ignoring a "fatal" error 🤔
